themes/default/content/blog/using-pulumi-securely/index.md

```diff
@@ -0,0 +1,97 @@
+---
+title: "Using Pulumi Securely"
+
+date: 2022-10-12
+
+meta_desc: Using Pulumi Securely.  Organization Access Token, Team Access Token, Teams, and Managed Federated Identity
+
+meta_image: meta.png
+
+authors:
+    - tushar-shah
+
+tags:
+    - security
+    - pulumi-enterprise
+
+---
+
+Cloud computing’s greatest strength and weakness is the proliferation of a massive number of services globally. In order to adequately assess and mitigate the inherent risks for your company, your customers, and your employees, cloud architects are typically responsible for a massive surface area of potential endpoints and vectors of attack.
+
+<!--more-->
+
+## Cloud Security Posture
+
+AWS likes to talk about a [Shared Responsibility Model](https://aws.amazon.com/compliance/shared-responsibility-model/), their friendly way of saying, “we’ve got our stuff covered and if you’re compromised, it’s almost certainly your fault.”  They have built up practices and recommendations like the AWS Well Architected Framework; pillars of architecture philosophy that give you your best chances to securely use their resources (following these recommendations is easy with [Crosswalk for AWS]({{< relref "/docs/guides/crosswalk/aws" >}}) from Pulumi).  Increasing high profile attacks against private and open source software supply chains has developers increasingly on high alert and conscious of incorporating more security into their devops.
+
+Risks associated with maintaining a good cloud security posture extend to your infrastructure as code with Pulumi and it’s important to leverage the features and capabilities of the Pulumi platform along with good industry practices to secure your code, configuration and secrets.
+
+## Pulumi’s Hierarchy
```

Comment:
Can I suggest re-ordering this section to be Org -> Team -> User (biggest to smallest). It's also the way that you ordered them [here](https://github.com/pulumi/pulumi-hugo/pull/1996/files#diff-e837a64db5f535174a06d7c5120653b1583b064ae20281c893c072ce94f84e18R69)

Reply:
Re-ordered them